### PR TITLE
High: pgsql: Support PostgreSQL 10

### PR DESCRIPTION
**Pull request by ytakeshita**

### CODE REVIEW

File: heartbeat/pgsql [hunk 1]
```diff
@@ -593,7 +597,7 @@ pgsql_real_start() {
     pgctl_options="$pgctl_options -o '$postgres_options'"
 
     # Invoke pg_ctl
-    runasowner "unset PGUSER; unset PGPASSWORD; $OCF_RESKEY_pgctl $pgctl_options start"
+    runasowner "unset PGUSER; unset PGPASSWORD; $OCF_RESKEY_pgctl $pgctl_options -W start"
```
Comment (krig) [inline heartbeat/pgsql]: Is the -W flag backwards compatible?

Comment (ytakeshita) [reply heartbeat/pgsql]: Yes, it is for backwards compatible.
The default action of "pg_ctl start" and "pg_ctl promote" is changed on PostgreSQL 10.
The default is
PostgreSQL 9.x : no wait (-W)
PostgreSQL 10 : wait (-w)

Comment (krig) [reply heartbeat/pgsql]: Makes sense, thank you!

**Comment by ytakeshita** [comment]
Thank you for merging !

**Pull request merged**